Automattic/pocket-casts-android
# Playback 2023 - Add YoY story animation

## Description
This adds YoY story animation (pdeCcb-3NF-p2)

## Testing Instructions
1. Log in to an account that has a few episodes listened this year and last year
3. Go to Profile and tap the End of Year card
4. Navigate to the Year over Year story
5. ✅ The bars should animate and the text should fade-in
6.  Share this story
7. ✅ Story image should be shown correctly

## Screenshots or Screencast 

https://github.com/Automattic/pocket-casts-android/assets/1405144/960e15f1-f9ba-4683-80d8-1a71c05c5037




## Checklist
 
#### I have tested any UI changes...
